.github/workflows/doc.yml

```diff
@@ -0,0 +1,49 @@
+name: Docs
+on:
+  push:
+    paths:
+      - 'doc/'
```

Comment:
After merging, I noticed that GitHub Actions were not responding to changes under doc/.
I fixed this in a PR #108  as I needed to specify the wildcard `**`.

https://docs.github.com/en/actions/writing-workflows/workflow-syntax-for-github-actions#onpushpull_requestpull_request_targetpathspaths-ignore

Reply:
Thanks for fixing it!